app/controllers/reservations_controller.rb

```diff
@@ -14,6 +14,8 @@ def set_rights
   end
 
   def index
+#add method here so everyone can see the occupied time slots
+  	muzikusrezervation
```

Comment:
Bu methodu buraya eklemek nasil herkesin occupied time slot'u gormesini sagliyor?

Kodundan gordugum kadariyla, persembe ve sali gitar dersi vs. var mi diye check ediyorsun eger yoksa ekliyorsun, her bu sayfa yuklendiginde. Asil anlamadigim, bu islemin `so everyone can see the occupied time slots` ile alakasi.